lib/util.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+
+const cli = require('heroku-cli-util')
+
+exports.starterPlan = a => !!a.plan.name.match(/(dev|basic)$/)
+
+exports.configVarNamesFromValue = (config, value) => {
+  const sortBy = require('lodash.sortby')
+
+  let keys = []
+  for (let key of Object.keys(config)) {
+    if (config[key] === value) keys.push(key)
+  }
+  return sortBy(keys, k => k !== 'DATABASE_URL', 'name')
+}
+
+exports.databaseNameFromUrl = (uri, config) => {
+  const url = require('url')
+
+  let names = exports.configVarNamesFromValue(config, uri)
+  let name = names.pop()
+  while (name === 'DATABASE_URL') name = names.pop()
```

Comment:
This should probably be the subject of another PR, but I noticed this behavior breaks when `DATABASE_URL` is the only attachment:


```
maciek@mothra:~$ h pg:info --app young-ridge-18903
=== 
Plan:        Hobby-dev
Status:      Available
Connections: 0/20
PG Version:  9.5.4
Created:     2016-11-01 18:56 UTC
Data Size:   7.4 MB
Tables:      0
Rows:        0/10000 (In compliance)
Fork/Follow: Unsupported
Rollback:    Unsupported
Add-on:      postgresql-infinite-56908

maciek@mothra:~$ h addons --app young-ridge-18903

Add-on                                         Plan       Price  State  
─────────────────────────────────────────────  ─────────  ─────  ───────
heroku-postgresql (postgresql-infinite-56908)  hobby-dev  free   created
 └─ as DATABASE
```

(note the empty name after the `===` header in the `pg:info` output)